src/Lecture1.hs

```diff
@@ -119,4 +134,8 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Integer -> [Integer] -> String
+lowerAndGreater n list = show n ++ " is greater than " ++ show lw ++ " elements and lower than " ++ show gr ++ " elements"
+    where
+        gr = length $ filter ( > n) list
+        lw = length $ filter ( < n) list  
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
With the recursive solution and manual check on every element in the list on both conditions at the same time (without `filter` function), you would be able to write it more performant with only one traversal of the list 🙂

Reply:
you mean tail - recursion with two aggregates ?